pages/_en-US/twilightmenu/custom-dsi-3ds-skins.md

```diff
@@ -57,17 +53,19 @@ These must be PNG files. Transparency is supported, however only 100% transparen
 | batterylow         | Used when in DS mode                                                        |
 
 ### Paletted textures (`grf` folder)
-These must be 4 BPP (16 color) BMP files files.
+These must be 4 BPP (16 color) BMP files.
 
-To edit these in GIMP, select `Image` -> `Mode` -> `RGB` to allow changing colors, then when done changing colors, select `Image` -> `Mode` -> `Indexed...` to convert back to paletted. When switching to indexed, ensure that `Generate optimum palette` is checked and `Maximum number of colors` is set to `16`.
+Editing these in GIMP requires the image to be set in "RGB" mode, toggleable via the setting in `Image` -> `Mode` -> `RGB`. When done, select `Image` -> `Mode` -> `Indexed...` to convert back to paletted.
 
-**Note:** Some images in the DSi theme have their palettes overridden based on the user's profile color. If editing the colors of these ensure that the `UserPalette` option for it in the `theme.ini` is set to `0`.
+Indexed Images requires an exact count of 16 colors, with the first of which (#0) being transparent (#FF00FF).
+- To set the transparent color as the first entry in the colormap, right click the colormap and select `Rearrange Colormap...`. Then, simply move the transparent color to the slot and select `OK`
+- If there are over 16 colors, set `Maximum number of colors` to 16
```

Comment:
Looking it over again you should probably mention that this is when converting to Indexed mode, the others of these bullets are talking about the colormap dialog. Maybe just add this to line 58 above since there's no reason not to set the max colors to 16.